### DISCUSSION TITLE
Yarn to PNPM within monorepo setup, 'module not found' error packages

### DISCUSSION THREAD

**Discussion by Muhammad-Hashim** [pnpm/pnpm#7816] [General]
I'm encountering an issue while transitioning from Yarn to PNPM within my monorepo setup, particularly with a 'module not found' error. Here's the breakdown:

Issue Description:
In our monorepo setup, we define new components within packages. However, upon transitioning to PNPM, we're encountering a 'module not found' error specifically related to these newly defined components.

Request for Assistance:
Could you please help address this error so that our transition to PNPM can proceed smoothly? Once resolved, I'll prepare a pull request for the changes.

**Comment by bzitzow-osh** [comment]
Describe more details about your setup: Your package structure, dependency structure, etc.
